app/core/src/main/java/com/fsck/k9/Identity.kt

```diff
@@ -0,0 +1,55 @@
+package com.fsck.k9
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class Identity(
+        val description: String? = null,
+        val name: String? = null,
+        val email: String? = null,
+        val signature: String? = null,
+        val signatureUse: Boolean = false,
+        val replyTo: String? = null
+) : Parcelable {
+    override fun toString(): String {
```

Comment:
data classes provide a usable `toString()` method. Is there a reason we're keeping the old one?

Reply:
Nope